src/main/kotlin/app/revanced/patcher/PatcherContext.kt

```diff
@@ -27,35 +23,31 @@ data class PatcherContext(
          * Merge integrations.
          * @param logger A logger.
          */
-        fun merge(logger: Logger) {
-            with(context.bytecodeContext.classes) {
+        fun merge(logger: Logger, dexFileNamer: DexFileNamer) {
+            context.bytecodeContext.classes.apply {
                 for (integrations in integrations) {
-                    callback?.let { it(integrations) }
+                    logger.info("Merging $integrations")
 
-                    for (classDef in lanchon.multidexlib2.MultiDexIO.readDexFile(
-                        true,
-                        integrations,
-                        NAMER,
-                        null,
-                        null
-                    ).classes) {
+                    for (classDef in lanchon.multidexlib2.MultiDexIO.readDexFile(true, integrations, dexFileNamer, null, null).classes) {
```

Comment:
Import `MultiDexIO` instead of qualify, since you've done that for `DexFileNamer` as well.

Reply:
Done.